src/Microsoft.AspNetCore.SignalR.Core/DefaultHubLifetimeManager.cs

```diff
@@ -98,18 +113,20 @@ public override Task SendAllAsync(string methodName, object[] args)
             return Task.WhenAll(tasks);
         }
 
-        private Task SendAllWhere(string methodName, object[] args, Func<HubConnectionContext, bool> include)
+        private void SendToGroupConnections(string methodName, object[] args, IEnumerable<HubConnectionContext> connections, Func<HubConnectionContext, bool> include, ref List<Task> tasks, ref SerializedHubMessage message)
```

Comment:
Why IEnumerable? Don't we have a List or something more concrete?

Reply:
There is an `ICollection<>` from the group's `IDictionary<>.Values` but it would just be enumerated over anyway.

Reply:
Yea, that structure we have is garbage.